# Topics
depth_topic: "/camera/depth/image_rect_raw"
landing_zone_topic: "/landing_zone_detection/landing_zone"

# IDR
baseline: 95            # mm
hfov: 1.5708            # 90 +- 1 degrees
vfov: 1.13446           # 65 +- 1 degrees
dfov: 1.55334           # 89 +- 1 degrees
padding_multiplier: 0.5   # 3x